osc/util/debquery.py

```diff
@@ -86,6 +72,40 @@ def read(self, all_tags=False, self_provides=True, *extra_tags):
         self.__parse_control(control, all_tags, self_provides, *extra_tags)
         return self
 
+    def __open_tar(self, arfile):
```

Comment:
I'm not sure if the leading double underscores are a good thing, because they lead to name mangling.
From https://peps.python.org/pep-0008/ :
```
__double_leading_underscore: when naming a class attribute, invokes name mangling
(inside class FooBar, __boo becomes _FooBar__boo).
```

Single leading underscores should be just fine.